docker-compose.override.yml

```diff
@@ -0,0 +1,6 @@
+version: '3'
```

Comment:
why is this extra override needed? and does this mean the service will always run on 8080 instead of 80 now?

Reply:
If you want to spawn more instances of the same service, they must use ephemeral ports as otherwise they would clash on 8080 or 80.
By default then you run docker-compose without any params it will use `docker-compose.yml` and `docker-compose.override.yml` if any exists in the same folder.

If you run just docker-compose it would run the service withouht LB and expose it on 8080 (I'll change that to 80)
and if you run with `-f docker-compose.lb.override.yml` will run the LB on 80 pointing to all service instances internal 8080

Reply:
I've changed port. Now service internally runs on 8080 and docker exposes that on 80 in single instance or on LB